### PR TITLE
Create: Landing Page and Navbar

### PR DESCRIPTION
**Pull request by Dimpal-Kalita**

### CODE REVIEW

File: src/Components/Landing/Images/Background.png
Comment (codadept) [inline src/Components/Landing/Images/Background.png:1]: Create a public directory in the root along-side src. And also create an `assets` directory inside public.

Move these images from `src/Components/Landing/Images` to `public/assets/` and then you can access the images in your HTML as

Suppose you want to use the image in the location `public/assets/background.png`, then 

```html
<img src="/assets/background.png" alt="background"/>
```

Do these for all the images in `src/Components/Landing/Images`

> Note: No need to write `public` as vite serves public as root `/`

File: src/Components/Landing/Landing.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
```
Comment (codadept) [inline src/Components/Landing/Landing.jsx]: No need to import React since release of `React 17`

File: src/Components/Landing/landing.module.scss
Comment (codadept) [inline src/Components/Landing/landing.module.scss:1]: Don't use any absolute units like `px, pt, cm` etc. Only use relative units like `vh, vw, rem, %` etc.

File: src/Components/Landing/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <div className={style.boatR}>
+        <img src={boatright} alt="" />
+      </div>
+
+      <div className={style.birdL}>
+        <img src={birdLeft} alt="" />
+      </div>
+
+      <div className={style.birdR}>
+        <img src={birdRight} alt="" />
```
Comment (codadept) [inline src/Components/Landing/Landing.jsx]: Provide appropriate alt text to all the images.

File: src/Components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+        <Link to="/Gallery">
+          <li>Gallery</li>
+        </Link>
+        <Link to="/Team">
+          <li>Team</li>
+        </Link>
+        <Link to="/Sponser">
+          <li>Sponser</li>
+        </Link>
+        <Link to="/Artists">
```
Comment (codadept) [inline src/Components/Navbar/Navbar.jsx]: Lower case all the links `/artists`, `/sponsor`, `/team`, `/gallery`

File: src/Pages/Home/Home.jsx [hunk 4]
```diff
@@ -1,12 +1,10 @@
-import { Button } from "../../Components";
-
 import style from "./Home.module.scss";
+import Landing from "../../Components/Landing/Landing";
```
Comment (codadept) [inline src/Pages/Home/Home.jsx]: Follow the template properly. Import and export the component in `src/Components/index.jsx` as 

```jsx
import Button from "./Button/Button";
import Landing from "./Landing/Landing";

export {Button, Landing};
```

and then you can use it here as 

```jsx

import { Landing } from "../../Components";
```

File: src/Components/Navbar/Navbar.jsx
Comment (codadept) [inline src/Components/Navbar/Navbar.jsx:1]: Use the Navbar in App.jsx

**Review by codadept — CHANGES_REQUESTED** [review]

**Comment by codadept** [comment]
1) Eslint errors and warnings

image

When you will run `pnpm lint` (which is also automatically set as recommit hook), you may get errors or warnings (like above). Your task is to fix those errors or warnings before pushing the code.

2) Positioning issues

image

Basically don't use pixels or other absolute units as it would seem fine in your PC, but maybe broken in someone else's resolution. Also don't center stuffs using left right property. Fix this issue, as you can see the eyes are not in the center of the website in my device.

3) Also the background image is stretched

Instead of using `background-size` property, use these properties (understand why am I telling you to use this)

image

Also once you have shifted the images to `public/assets/background.png` then new url would be `background-image : url("/assets/background.png")`

4) No need to compile the scss file

image

No need to compile the scss files to css, map.css files. They are internally done by vite. So delete all the .css and .map.css files you have created. 

image

Also you need to import the `Landing.module.scss` file instead of the css file.

Also try to follow the convention of file names :) Checkout other .scss file names

If the component file name is `Landing.jsx` then it's scss file name would be `Landing.module.scss` with a capital `L`. Follow the conventions properly

File: src/Components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -1,5 +1,36 @@
+import React from "react";
```
Comment (codadept) [inline src/Components/Navbar/Navbar.jsx]: Remove the React import. Its redundant

**Review by codadept — CHANGES_REQUESTED** [review]

**Review by username-naz — APPROVED** [review]

**Pull request merged**